Alpaca.Markets/Messages/JsonHistoricalItems.cs

```diff
@@ -28,6 +28,9 @@ internal class JsonHistoricalItems<TApi, TJson> : IHistoricalItems<TApi> where T
         [JsonProperty(PropertyName = "resultsCount", Required = Required.Default)]
         public Int64 ResultsCount { get; set; }
 
+        [JsonProperty(PropertyName = "db_latency", Required = Required.Default)]
+        public Int64 DbLatency { get; set; }
```

Comment:
Why do we need this property? It's not visible through the `IHistoricalItems` interface and therefore not available to clients. If we want to expose this property we have to add it into the appropriate interface too.